### ISSUE TITLE
VMDK Image not start OS

### ISSUE THREAD

**Issue by Saentist** [home-assistant/operating-system#1056]
**Hardware Environment**
- [x] OVA (VMWARE)

**Home Assistant OS release:**
- [x] Fresh installation of release 4.16
```https://github.com/home-assistant/operating-system/releases/download/4.16/hassos_ova-4.16.vmdk.gz```

**Description of problem:**
Starting stop on this point all time. I'm using VMWare Workstation 16
![image](https://user-images.githubusercontent.com/16975883/101095748-127edc80-35c7-11eb-9ee6-89d5e1f65626.png)

**Comment by thfrlw** [comment]
I've just run into pretty much the same issue

**Comment by agners** [comment]
Hm, it seems the Linux kernel doesn't see the harddrive. What type of disk controller are you emulating in this machine configuration?

**Comment by Saentist** [comment]
> Hm, it seems the Linux kernel doesn't see the harddrive. What type of disk controller are you emulating in this machine configuration?

SCSI with is default.
In manual is not specified with must be selected.

```
For Vmware Workstation create a new virtual machine, select “Custom”, 
make it compatible with the default of Workstation and ESX, 
Choose “I will install the operating system later”, 
select “Linux” -> “Other Linux 5.x or later kernel 64-bit”, 
give it at least 2 GB RAM and 1vCPU, 
select “Use Bridged Networking” then “Use an existing virtual disk” and select the VMDK file above, 
after creation of VM go to “Settings” and “Options” then “Advanced” and select “Firmware type” to “UEFI”.```

**Comment by Saentist** [comment]
Image is for **SATA**
Manual need update, or better export of virtual mashine in place of hdd image.

**Comment by agners** [comment]
@Saentist so it worked with SATA? Yeah I think we are missing a SCSI driver or two which are common in virtualization. But if SATA works fine, no problem using it.

**Comment by Saentist** [comment]
Yes work with SATA mode connected disk image.
Manual do not say nothing about disk connecting mode.

**Comment by jxbrown** [comment]
I just updated HASSIO from 4.16 to 5.8 on VMWare workstation. The update went fine but after I rebooted I get the following error.

![image](https://user-images.githubusercontent.com/50732522/102457958-65f61f00-3ff8-11eb-8f2c-b2b5fbfe8006.png)

I'm using a SATA HD configuration in my settings.  Easily reproducible. 

![image](https://user-images.githubusercontent.com/50732522/102458268-da30c280-3ff8-11eb-98a1-77ebeda94a46.png)
